sanic/pages/styles/DirectoryPage.css

```diff
@@ -0,0 +1,67 @@
+#breadcrumbs a:hover {
+    text-decoration: underline;
+}
+
+#breadcrumbs .path-0 a {
+    text-decoration: none;
+}
+
+#breadcrumbs span::after {
+    content: "/";
+    text-decoration: none;
+    padding: 0 0.25em;
+}
+
+#breadcrumbs .path-0 a::before {
+    content: "🏠";
+}
+
+#breadcrumbs>span>a {
+    color: #ff0d68;
+}
+
+main a {
+    color: inherit;
+    font-weight: bold;
+}
+
+table.autoindex {
+    width: 100%;
+    font-family: monospace;
+}
+
+table.autoindex tr {
+    display: flex;
+}
+
+table.autoindex tr:hover {
+    background-color: #ddd;
+}
```

Comment:
Consider using #8884 (transparency) and avoid the media selector altogether. Otherwise, keep the default and the selector adjacent in the CSS code whenever conveniently possible.